grakn-graph/src/main/java/ai/grakn/factory/SystemKeyspace.java

```diff
@@ -221,4 +210,35 @@ private static void checkVersion(GraknGraph graph){
             throw GraphOperationException.versionMistmatch(existingVersion);
         }
     }
+
+    /**
+     * Loads the system ontology inside the provided grakn graph.
+     *
+     * @param graph The graph to contain the system ontology
+     */
+    private static void loadSystemOntology(GraknGraph graph){
+        //Keyspace data
+        ResourceType<String> keyspaceName = graph.putResourceType("keyspace-name", ResourceType.DataType.STRING);
+        graph.putEntityType("keyspace").key(keyspaceName);
+
+        //User Data
+        ResourceType<String> userName = graph.putResourceType("user-name", ResourceType.DataType.STRING);
+        ResourceType<String> userPassword = graph.putResourceType("user-password", ResourceType.DataType.STRING);
+        ResourceType<String> userPasswordSalt = graph.putResourceType("user-password-salt", ResourceType.DataType.STRING);
+        ResourceType<String> userFirstName = graph.putResourceType("user-first-name", ResourceType.DataType.STRING);
+        ResourceType<String> userLastName = graph.putResourceType("user-last-name", ResourceType.DataType.STRING);
+        ResourceType<String> userEmail = graph.putResourceType("user-email", ResourceType.DataType.STRING);
+        ResourceType<Boolean> userIsAdmin = graph.putResourceType("user-is-admin", ResourceType.DataType.BOOLEAN);
+
+        graph.putEntityType("user").key(userName).
+                resource(userPassword).
+                resource(userPasswordSalt).
+                resource(userFirstName).
+                resource(userLastName).
+                resource(userEmail).
+                resource(userIsAdmin);
+
+        //System Version
+        graph.putResourceType("system-version", ResourceType.DataType.STRING);
```

Comment:
There seems to be a lot of stuff missing here that was in the graql file. Is it intentional?